docs/user-guide/glossary.md

```diff
@@ -185,12 +185,18 @@ quantified expression
   details about quantifiers in CVL.
 
 receiveOrFallback
-  A special function we introduce in every contract to model the behavior of solidity
+  A special function we introduce in every contract to model the behavior of Solidity
   for calls with no data or that do not resolve to any contract function.
   It will call the receive function if present for calls with no data, and otherwise the fallback function.
   Shows up in the parametric rules or invariants, as well as in the call trace for such calls, written `<receiveOrFallback>()`.
   See also [Solidity Documentation](https://docs.soliditylang.org/en/latest/contracts.html#fallback-function).
 
+rule name pattern
+  Rule names, like all CVL identifiers, have the same format as Solidity identifiers: they consist of a combination of letters, digits,
+  dollar signs and underscores, but cannot start with a digit (see [here](https://docs.soliditylang.org/en/v0.8.16/path-resolution.html#allowed-paths)).
+  In addition, rule name patterns can include the wildcard `*` that can replace any sequence of valid identifier characters.
```

Comment:
maybe need to clarify wildcards only apply to option values not for example to spec files

Reply:
Added a clarification